src/components/Spotlight/Spotlight.tsx

```diff
@@ -129,15 +161,33 @@ const Spotlight = ({
       // (if the annotation is below the target, this should be an addition, not subtraction)
       const offset = annotationHeight ? newTargetTop - annotationHeight : newTargetTop;
 
-      window.scrollBy({
-        top: offset,
-        left: 0,
-        behavior: !animateTargetChanges || prefersReducedMotion ? 'auto' : 'smooth',
-      });
+      if (scrollTarget.current) {
+        // TODO:
```

Comment:
Is this still a TODO or has it been TODone?

Reply:
Still a TODO, I'll make a ticket

Reply:
Done 👍